packages/sdk/src/IntrospectionEngine.ts

```diff
@@ -269,7 +269,12 @@ export class IntrospectionEngine {
           const binaryPath = await resolveBinary(BinaryType.introspectionEngine)
           debugRpc('starting introspection engine with binary: ' + binaryPath)
 
-          // TODO Why does the engine need the cwd
+          // about `cwd`
```

Comment:
Would it make sense to move this comment near the `cwd: this.cwd,` line or would the formatting be weird then?

Reply:
moved ✅